### PR TITLE
Update LIP 0048 

### PR DESCRIPTION
**Pull request by mjerkov**
Updated according to [Trello card](https://trello.com/c/RqoOpXeM/1432-small-issues-in-lips):

- Use b.header.generatorAddress instead of b.generatorPublicKey.
- There are two events in the LIP, but in the constants table there is just one event.
- The type CCM is used but not defined.

Comment: No problem with events was found - there are three events and all of them are mentioned in the Constants table.

### CODE REVIEW

File: proposals/lip-0048.md [hunk 1]
```diff
@@ -39,28 +39,29 @@ As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater
...
+| ------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
 | `Transaction` | object | Must follow the `transactionSchema` schema defined in [LIP 0068](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0068.md) with the only difference that `params` property is not serialized and contains the values of parameters of `paramsSchema` for the corresponding transaction. | An object representing a non-serialized transaction. |
+| `CCM`         | object | Must follow the `crossChainMessageSchema` schema defined in [LIP 0049](lip-0049#ccmschema).                                                                                                                                                                                                   | The type of cross-chain messages.                    |
```
Comment (sergeyshemyakov) [inline proposals/lip-0048.md:45]: The link to the schema is broken. I suspect that `()` should be changed into `[]`. 
It looks like the link is resolved into the relative path `/lip-0049` from the location of the file, which is interesting.

**Review by sergeyshemyakov — APPROVED** [review]
Good catch with `=` in schemas! Also judging from the git history, the events were fixed by Maxime sometime in December.
Just one remark from my side: make sure that the link works before merging.

**Review by gkoumout — CHANGES_REQUESTED** [review]
Good job Miroslav! I just caught one inconsistency which was here even before but we had not noticed, plz have a look.

File: proposals/lip-0048.md [hunk 2]
```diff
@@ -39,28 +39,29 @@ As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater
 
 We use the definition of the following type:
 
-| Name | Type | Validation | Description |
-|------|------|------------|-------------|
+| Name          | Type   | Validation                                                                                                                                                                                                                                                                                    | Description                                          |
```
[the remark this answers is not in the document]
Comment (mjerkov) [reply proposals/lip-0048.md:42]: Thanks for noticing this! Definitely something to be corrected (please check the next commit).

File: proposals/lip-0048.md [hunk 3]
```diff
@@ -35,32 +35,32 @@ As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater
...
+| `EVENT_NAME_INSUFFICIENT_FEE`        | string | "insufficientFee"                   | Event name of the InsufficientFee event.                          |
+| `LENGTH_ADDRESS`                     | uint32 | 20                                  | The length of an address in bytes.                                |
+| `LENGTH_TOKEN_ID`                    | uint32 | 8                                   | The length of a token ID in bytes.                                |
+| **Configurable Constants**           |        | **Mainchain Value**                 |                                                                   |
+| `MIN_FEE_PER_BYTE`                   | uint64 | 1000                                | Minimum amount of fee per byte required for transaction validity. |
+| `TOKEN_ID_FEE`                       | bytes  | `TOKEN_ID_LSK = 0x0000000000000000` | Token ID of the token used to pay the transaction fees.           |
+
+### Type Definitions
+
+We use the definition of the following type:
```
Comment (gkoumout) [inline proposals/lip-0048.md:57]:
```suggestion
We use the definition of the following types:
```

**Review by gkoumout — APPROVED** [review]
Approved, just mentioned a small typo. Bravo!

File: proposals/lip-0048.md (hunk 3 above)
Comment (mjerkov) [reply proposals/lip-0048.md]: Thanks again, my mistake this time :)!

**Review by karmacoma — APPROVED** [review]

**Pull request merged**